Library/Homebrew/utils/free_port.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Utils
+  def self.free_port
```

Comment:
How about something like `free_port_number` or similar?

Reply:
Don't like it really. The word "port" is well defined and not ambiguous in our context, so `port_number` sounds like a tautology

Reply:
Cool 👍